.github/workflows/publish-packages-1.0.yml

```diff
@@ -23,6 +23,12 @@ jobs:
 
   build-pack-publish:
     runs-on: windows-latest
+    permissions:
+      contents: read
```

Comment:
nit: I don't think `contents: read` is needed here. By default GITHUB_TOKEN has read:

https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Thanks for the feedback. Yes, read should be to have default with read, here since I I overwrite the permissions with `id_token`, seems I need specify the read permission explicitly (I added when adding `id_token` alone didn't work). But I can give it another try.